fabric-input-events-v1/src/main/java/net/fabricmc/fabric/mixin/event/input/client/KeyboardMixin.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.event.input.client;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.client.Keyboard;
+
+import net.fabricmc.fabric.impl.client.input.InputCallbacksImpl;
+
+@Mixin(Keyboard.class)
+public class KeyboardMixin {
+	@Inject(method = "method_22678(JIIII)V", at = @At("HEAD"))
```

Comment:
Could you please leave a comment saying which method owns this synthetic method in the signature for all of these across keyboard and mouse?